### ISSUE TITLE
Horizontal tabs keep reappearing when I switch windows/applications

### ISSUE THREAD

**Issue by FarrenHawk** [nuclearcodecat/shimmer#37]
Currently using Firefox on Hyprland/Arch (not sure if that's helpful) and I keep running into an issue where certain actions cause the title bar to spontaneously materialize back into existence, such as switching desktops, switching applications or sometimes even switching tabs. The preface value in sideberry is normal.

I have found that I can temporarily fix this by toggling the `Add preface to the browser's window title if Sidebery sidebar is active` switch, or by re-uploading the addon data json file to sidebery. 

I don't know jack about coding, but I wonder if this has something to do with the way my WM handles window titles, and maybe the preface value is lost each time the window title is lost (i.e. i switch to a different application) but that's just a theory.

One thing that kind of fixes it is changing `shimmer.native-vertical-tabs` to `true`, but that causes the red close window to exist by itself in the horizontal tab bar (and it snaps to the left)

I can provide a screenshot or screencap if needed but im lazy rn.

Also, is it possible to hide sidebery until i want to access my tabs? like if i hover over the left side of the screen it opens? :3

**Comment by nuclearcodecat** [comment]
yeah this has to be something with your wm... i'll check on my laptop later, but it's too old for wayland lol, so it probably won't repro. you could try reinstalling firefox (idk if `pacman -Rn` takes care of ff profiles, but it might be worth to try removing that too. export your bookmarks beforehand). unfortunately i have to use the preface value thingy because `#navigator-toolbox` (which contains tabs etc.) is outside of `#browser`... if it appeared after, i could apply styles based on the `#sidebar-box` being `:not([hidden])`... anyhow, try reinstalling, tell me how it goes. you could also try installing some other wm and switching in your display manager

> Also, is it possible to hide sidebery until i want to access my tabs? like if i hover over the left side of the screen it opens? 
i was supposed to make this for a major update but i'm a lazy fuck.. xP it should be possible. if i will have some motivation i'll work on it

**Comment by nuclearcodecat** [comment]
hi i've been running hyprland on my laptop for the past few days and i don't have the issues you do. this might be your config or something else

**Comment by FarrenHawk** [comment]
ah gotcha. weird

**Comment by Michle123** [comment]
had this same thing happen to me and what seemingly fixed it was disabling some extensions i had. my assumption is that userchrome toggle extended caused the issue as i had that remaining from my shyfox install before

**Comment by KeePach** [comment]
I had the same issue, deactivated userchrome toggle extended as I also had it and the problem dissapeared so I guess is a compatibility issue with that extension.
